bitnami/nginx-intel/templates/deployment.yaml

```diff
@@ -193,76 +193,21 @@ spec:
           resources: {{- toYaml .Values.resources | nindent 12 }}
           {{- end }}
           volumeMounts:
-            {{- if or .Values.serverBlock .Values.existingServerBlockConfigmap .Values.ldapDaemon.enabled }}
+            {{- if or .Values.serverBlock .Values.existingServerBlockConfigmap }}
             - name: nginx-server-block-paths
               mountPath: /opt/bitnami/nginx/conf/server_blocks
```

Comment:
The `nginx-server-block-paths` was introduced as a consequence of LDAP addition. This means that it should also be removed and update the logic accordingly.

Refer to https://github.com/bitnami/charts/pull/2608/files#r427295257 for further details.